### PR TITLE
Verifications after chectl installation

### LINKED ISSUE #14428: [Doc] Add verification steps after chectl install
Please add verification steps after the installation of chectl in https://github.com/eclipse/che-docs/blob/master/src/main/pages/che-7/overview/proc_installing-the-chectl-management-tool-on-linux-or-macos.adoc -- something like:

```
. Run the following command to verify that the `chectl` binary that is used is `/usr/local/bin/chectl`:

----
which chectl
----

[TIP]
====
If the output of the `which chectl` command is not `/usr/local/bin/chectl`, it probably means that an old version of chectl is present in a place with higher priority in your $PATH. If this other version appear to be unnecessary, run the following command to delete it:

----
which chectl | rm
----
====


. Run the following command to verify that the version of chectl that is the expected one.

----
chectl --version
----
```

### PR DESCRIPTION
**Pull request by themr0c**
Add verification steps after the installation of chectl 

fix eclipse/che/issues/14428

### CODE REVIEW

**Comment by themr0c** [comment]
@benoitf please review!

File: src/main/pages/che-7/overview/proc_installing-the-chectl-management-tool-on-linux-or-macos.adoc [hunk 1]
```diff
@@ -14,9 +14,49 @@ This section describes how to install the chectl management tool on Linux or Mac
 . Run the following commands in the terminal:
 +
 ----
-# bash <(curl -sL  https://www.eclipse.org/che/chectl/)
+$ bash <(curl -sL  https://www.eclipse.org/che/chectl/)
```
Comment (rkratky) [inline src/main/pages/che-7/overview/proc_installing-the-chectl-management-tool-on-linux-or-macos.adoc:21]: Maybe use just `sh` to make it more universal?
```suggestion
$ sh <(curl -sL  https://www.eclipse.org/che/chectl/)
```

Comment (rkratky) [reply src/main/pages/che-7/overview/proc_installing-the-chectl-management-tool-on-linux-or-macos.adoc:21]: Ah, I see the script itself uses `bash`. Ok, disregard.

File: src/main/pages/che-7/overview/proc_installing-the-chectl-management-tool-on-linux-or-macos.adoc [hunk 2]
```diff
@@ -14,9 +14,49 @@ This section describes how to install the chectl management tool on Linux or Mac
 . Run the following commands in the terminal:
```
Comment (rkratky) [inline src/main/pages/che-7/overview/proc_installing-the-chectl-management-tool-on-linux-or-macos.adoc]:
```suggestion
. Run the following command in the terminal (this downloads and executes the `install.sh` script):
```

Comment (rkratky) [reply src/main/pages/che-7/overview/proc_installing-the-chectl-management-tool-on-linux-or-macos.adoc]: The prereqs for this procedure should include the necessity to have the `sudo` command (environment) configured for the current user. (This isn't configured by default on RHEL, for example.)

File: src/main/pages/che-7/overview/proc_installing-the-chectl-management-tool-on-linux-or-macos.adoc [hunk 3]
```diff
@@ -14,9 +14,49 @@ This section describes how to install the chectl management tool on Linux or Mac
...
 +
 ----
-# bash <(curl -sL  https://www.eclipse.org/che/chectl/)
+$ bash <(curl -sL  https://www.eclipse.org/che/chectl/)
+----
++
+[NOTE]
+====
+The installation script requires superuser access.
```
Comment (rkratky) [inline src/main/pages/che-7/overview/proc_installing-the-chectl-management-tool-on-linux-or-macos.adoc]: The above comment about the `sudo` prereq should make this admonition redundant.

File: src/main/pages/che-7/overview/proc_installing-the-chectl-management-tool-on-linux-or-macos.adoc [hunk 4]
```diff
@@ -14,9 +14,49 @@ This section describes how to install the chectl management tool on Linux or Mac
...
+If necessary the user will be prompted for his password by sudo.
```
Comment (rkratky) [inline src/main/pages/che-7/overview/proc_installing-the-chectl-management-tool-on-linux-or-macos.adoc]: @themr0c, just a docs note: avoid 'if' and 'will' :)

File: src/main/pages/che-7/overview/proc_installing-the-chectl-management-tool-on-linux-or-macos.adoc [hunk 5]
```diff
@@ -14,9 +14,49 @@ This section describes how to install the chectl management tool on Linux or Mac
...
+====
++
+
+. Run the following command to verify that the `chectl` binary that is used is `/usr/local/bin/chectl`:
```
Comment (rkratky) [inline src/main/pages/che-7/overview/proc_installing-the-chectl-management-tool-on-linux-or-macos.adoc]:
```suggestion
. Run the following command to verify that the `chectl` binary used is `/usr/local/bin/chectl`:
```

File: src/main/pages/che-7/overview/proc_installing-the-chectl-management-tool-on-linux-or-macos.adoc [hunk 6]
```diff
@@ -14,9 +14,49 @@ This section describes how to install the chectl management tool on Linux or Mac
...
++
+----
+$ which chectl
+/usr/local/bin/chectl
+----
++
+[TIP]
```
Comment (rkratky) [inline src/main/pages/che-7/overview/proc_installing-the-chectl-management-tool-on-linux-or-macos.adoc]: This step should be before the installation of the right `chectl`:

1. (prereq) Remove all old or undesired versions of `chectl`
2. Run the install script
3. Verify that the install `chectl` is in your path

File: src/main/pages/che-7/overview/proc_installing-the-chectl-management-tool-on-linux-or-macos.adoc [hunk 7]
```diff
@@ -14,9 +14,49 @@ This section describes how to install the chectl management tool on Linux or Mac
...
+
+. Run the following command to verify that the version of chectl that is the expected one.
++
+----
+$ chectl --version
+----
++
+[TIP]
+====
+To determine the lates stable release navigate to the  link:https://github.com/che-incubator/chectl/releases[list of Chectl releases] and search for a release not containing "next" in her name.
```
Comment (rkratky) [inline src/main/pages/che-7/overview/proc_installing-the-chectl-management-tool-on-linux-or-macos.adoc]:
```suggestion
To identify the latest stable release, see the link:https://github.com/che-incubator/chectl/releases[list of chectl releases] and search for a release not containing "next" in the name.
```

File: src/main/pages/che-7/overview/proc_installing-the-chectl-management-tool-on-linux-or-macos.adoc [hunk 8]
```diff
@@ -14,9 +14,49 @@ This section describes how to install the chectl management tool on Linux or Mac
...
+====
+
+
+. Read the installation logs.
++
+----
+cat chectl-install.log
```
Comment (rkratky) [inline src/main/pages/che-7/overview/proc_installing-the-chectl-management-tool-on-linux-or-macos.adoc]:
```suggestion
$ cat chectl-install.log
```

**Review by rkratky — CHANGES_REQUESTED** [review]

**Comment by themr0c** [comment]
Sorry I messed up with the GitHub web interface. As an unfortunate consequence I had to rebase, reset-author, squash and use the force. I believe all your concerns are adressed @rkratky.

**Review by rkratky — APPROVED** [review]

**Comment by themr0c** [comment]
Thank you @rkratky !

**Pull request merged**